packages/engine/src/map/parseGeometry.ts

```diff
@@ -0,0 +1,67 @@
+import { Polygon, Vector3 } from "yuka";
```

Comment:
Though this is really nice code, if I'm not mistaken, we're going to need to generate the navmesh dynamically from map data. Anyway, how is this different from the NavMeshLoader from Yuka?

Reply:
this function is from that loader. i know that we need dynamic navmesh from map, but i needed graph to test.
it should be replaced